### PR TITLE
feat(charts): add analytics ID and KEY envs to provisioner deployment 

### PR DESCRIPTION
**Pull request by niladrih**

### CODE REVIEW

File: deploy/helm/charts/Chart.yaml [hunk 1]
```diff
@@ -4,10 +4,10 @@ description: Helm chart for OpenEBS Dynamic Local PV. For instructions to instal
 type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
-version: 4.1.0
+version: 4.2.0
```
Comment (avishnu) [inline deploy/helm/charts/Chart.yaml]: Shouldn't this be 4.1.1 ?

File: deploy/helm/charts/README.md [hunk 2]
```diff
@@ -68,14 +68,14 @@ helm install openebs-localpv openebs-localpv/localpv-provisioner --namespace ope
 
 | Parameter                                   | Description                                                                                                                                                                                 | Default                       |
 | ------------------------------------------- |---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-------------------------------|
-| `release.version`                           | LocalPV Provisioner release version                                                                                                                                                         | `4.1.0`                       |
+| `release.version`                           | LocalPV Provisioner release version                                                                                                                                                         | `4.2.0`                       |
```
Comment (avishnu) [inline deploy/helm/charts/README.md]: 4.1.1

File: deploy/helm/charts/Chart.yaml (hunk 1 above)
Comment (niladrih) [reply deploy/helm/charts/Chart.yaml]: Indeed, this is a patch release. Fixed.

File: deploy/helm/charts/README.md (hunk 2 above)
Comment (niladrih) [reply deploy/helm/charts/README.md]: Fixed.

**Review by avishnu — APPROVED** [review]
Lgtm

**Review by Abhinandan-Purkait — APPROVED** [review]

**Pull request merged**